lib/run_printer.py

```diff
@@ -161,13 +165,17 @@ async def dump_run(args):
 
     sleeper = BackoffSleeper(jitter=random.random())
     check_run = get_run_checker()
+    out_file = args.out_file.resolve() if args.out_file else None
```

Comment:
Please do this error checking in the argparser 🐳

Reply:
Resolved